pkg/gadgets/profile/block-io/tracer/tracer.go

```diff
@@ -119,19 +105,19 @@ func (t *Tracer) install() error {
 
 	blockRqCompleteLink, err := link.AttachRawTracepoint(link.RawTracepointOptions{Name: "block_rq_complete", Program: t.objs.IgProfioDone})
 	if err != nil {
-		return fmt.Errorf("error attaching tracing: %w", err)
+		return fmt.Errorf("attaching tracing block_rq_complete: %w", err)
```

Comment:
attaching tracing block_rq_complete -> attaching tracepoint block_rq_complete

?

Same in other locations.